accel-pppd/radius/radius.c

```diff
@@ -1006,6 +1017,52 @@ static int load_config(void)
 	return 0;
 }
 
+static void ev_radius_coa_vrf(struct ev_radius_t *ev)
+{
+	struct rad_attr_t *attr;
+
+	log_ppp_debug("ev_radius_coa_vrf started\n");
+
+	list_for_each_entry(attr, &ev->request->attrs, entry) {
+		int vendor_id = attr->vendor ? attr->vendor->id : 0;
+
+		if (vendor_id != Vendor_Accel_PPP)
+			continue;
+
+		if (attr->attr->id == Accel_VRF_Name) {
+			if (attr->len < 1) {
+				log_ppp_error("ev_radius_coa_vrf vrf attr len < 1\n");
+				ev->res = -1;
+				return;
+			}
+
+			log_ppp_debug("ev_radius_coa_vrf vrf attribute found value=%s\n", attr->val.string);
+
+			if (attr->len == 1 && attr->val.string[0] == '/') {
```

Comment:
doubts regarding default vrf selection, what if make it "default" or "none" or "0" (and allow empty as well)?
ex. if (attr->len == 0 || strcmp...).
also, with self-contained ap_session_set_vrf() the whole check is not necessary, just pass the value down to function